examples/agent-sandbox-rl/agent_sandbox_rl/strategies.py

```
@@ -0,0 +1,145 @@
+  results = [None] * len(fleet.tasks)
+  try:
+    for start in range(0, len(images), window):
+      batch = images[start:start + window]
+      for img in batch:
+        fleet.warm_image(img, replicas_override=replicas_override, wait=True)
+      batch_pairs = [(i, t) for img in batch for (i, t) in by_image[img]]
+      batch_tasks = [t for _i, t in batch_pairs]
+      logger.info("window [%d..%d): %d image(s), %d task(s)",
+                  start, start + len(batch), len(batch), len(batch_tasks))
+      batch_results = process_parallel(fleet, batch_tasks, process_fn, concurrency)
+      for (i, _t), r in zip(batch_pairs, batch_results):
```

Ruff (0.15.17)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)